Frontend/src/views/Login.vue

```diff
@@ -31,8 +31,9 @@
                     password: this.$data.password
                })
                .then(resp => {
-                   localStorage.email = this.email;
-                   localStorage.token = resp.data; 
+                   let respData = resp.data
+                   localStorage.setItem('token', respData)
```

Comment:
store the email in local as well, update password needs it

Reply:
Email should not be stored in localStorage. This should be fetched from the server.

Reply:
@julianpoy reason for that extra step?

Reply:
Not good practice - localStorage should not be used to store personal/identifying data.

Also not RESTful.